src/libraries/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -849,5 +854,145 @@ private static Complex Scale(Complex value, double factor)
+
+        //
+        // IMultiplicativeIdentity
+        //
+
+        /// <inheritdoc cref="IMultiplicativeIdentity{TSelf, TResult}.MultiplicativeIdentity" />
+        static Complex IMultiplicativeIdentity<Complex, Complex>.MultiplicativeIdentity => new Complex(1.0, 0.0);
+
+        //
+        // IMultiplyOperators
+        //
+
+        /// <inheritdoc cref="IMultiplyOperators{TSelf, TOther, TResult}.op_CheckedMultiply(TSelf, TOther)" />
+        static Complex IMultiplyOperators<Complex, Complex, Complex>.operator checked *(Complex left, Complex right) => left* right;
+
+        //
+        // INumberBase
+        //
+
+        /// <inheritdoc cref="INumberBase{TSelf}.One" />
+        static Complex INumberBase<Complex>.One => new Complex(1.0, 0.0);
+
+        /// <inheritdoc cref="INumberBase{TSelf}.Zero" />
+        static Complex INumberBase<Complex>.Zero => new Complex(0.0, 0.0);
+
+        //
+        // ISignedNumber
+        //
+
+        /// <inheritdoc cref="ISignedNumber{TSelf}.NegativeOne" />
+        static Complex ISignedNumber<Complex>.NegativeOne => new Complex(-1.0, 0.0);
+
+        //
+        // ISpanFormattable
+        //
+
+        /// <inheritdoc cref="ISpanFormattable.TryFormat(Span{char}, out int, ReadOnlySpan{char}, IFormatProvider?)" />
+        public bool TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan<char> format, IFormatProvider? provider)
+        {
+            int charsWrittenSoFar = 0;
+
+            // We have at least 6 more characters for: (0, 0)
+            if (destination.Length < 6)
+            {
+                charsWritten = charsWrittenSoFar;
+                return false;
+            }
+
+            destination[charsWrittenSoFar++] = '(';
+
+            bool tryFormatSucceeded = m_real.TryFormat(destination.Slice(charsWrittenSoFar), out int tryFormatCharsWritten, format, provider);
+            charsWrittenSoFar += tryFormatCharsWritten;
+
+            // We have at least 4 more characters for: , 0)
+            if (!tryFormatSucceeded || (destination.Length < (charsWrittenSoFar + 4)))
+            {
+                charsWritten = charsWrittenSoFar;
+                return false;
+            }
+
+            destination[charsWrittenSoFar++] = ',';
+            destination[charsWrittenSoFar++] = ' ';
+
+            tryFormatSucceeded = m_imaginary.TryFormat(destination.Slice(charsWrittenSoFar), out tryFormatCharsWritten, format, provider);
+            charsWrittenSoFar += tryFormatCharsWritten;
+
+            // We have at least 1 more character for: )
+            if (!tryFormatSucceeded || (destination.Length < (charsWrittenSoFar + 1)))
+            {
+                charsWritten = charsWrittenSoFar;
+                return false;
+            }
+
+            destination[charsWrittenSoFar++] = ')';
+
+            charsWritten = charsWrittenSoFar;
+            return true;
+        }
```

Comment:
Could the whole implementation be:
```C#
destination.TryWrite($"({m_real}, {m_imaginary}", out charsWritten);
```

Reply:
Ah, I guess the Complex applies the format to each value as well?

Reply:
Right. The provider/format is used to control how `real` and `imaginary` are printed.

If there's some "simpler" pattern here I'm not thinking of, then I'm all ears.

Reply:
If format wasn't needed or was predetermined rather than dynamic, it could be as simple as what I wrote above.  But interpolation doesn't support dynamic formats, so I don't have a simpler suggestion.